package.json

```diff
@@ -21,14 +21,14 @@
     "package": "node script/package",
     "clean:tslint": "rimraf tslint-rules/*.js",
     "compile:tslint": "tsc -p tslint-rules",
-    "lint": "npm run compile:tslint && tslint \"./app/{src,test}/**/*.{ts,tsx}\"",
-    "check-prettiness": "node script/is-it-pretty",
+    "lint": "npm run compile:tslint && npm run tslint && npm run eslint-check && npm run eslint",
+    "tslint": "tslint ./tslint-rules/*.ts ./app/{src,typings,test}/**/*.{ts,tsx}",
+    "eslint": "ts-node script/eslint.ts",
+    "eslint-check": "eslint --print-config .eslintrc.* | eslint-config-prettier-check",
     "publish": "node script/publish",
     "clean-slate": "rimraf out node_modules app/node_modules && npm install",
     "rebuild-hard:dev": "npm run clean-slate && npm run build:dev",
-    "rebuild-hard:prod": "npm run clean-slate && npm run build:prod",
-    "prettier:base": "prettier --single-quote --trailing-comma es5 --no-semi --write",
```

Comment:
I’ve 🔥’d them because the `prettier/prettier` rule in `.eslintrc.yml` does the same thing, and it looks nicer IMO. Instead of listing out all of the files checked, it only shows ones that need fixing, and it gives explicit details about what needs to be fixed.